### DISCUSSION TITLE
16Bit (RBG565) RBG Picker

### DISCUSSION THREAD

**Discussion by kitecraft** [Bodmer/TFT_eSPI#1203] [General]
Hello TFT community.
Just wondering how you choose and pick the colors you want to use with TFT_eSPI?   

I've been using this site here:  https://ee-programming-notepad.blogspot.com/2016/10/16-bit-color-generator-picker.html
But, I just came across this page: https://chrishewett.com/blog/true-rgb565-colour-picker/
Not quite as easy to use, but more fun to play with.

Cheers

**Comment by bobwolff68** [comment]
Yeah I have used those sites and they are ok from a "mechanical" perspective. They give you what you require. But I'm not an artsy person ... I know good color choices when I see them but I don't choose them so well. 

Here's the most lovely site I found - you can use the 'picker' to choose a color through sliders and such. But the MAGIC is clicking on "Pallet" and having them give you a full set of colors that compliment each other for a fantastic user interface.

https://ankiewicz.com/color/palette/00cccc

And while the site does NOT give you 565 colors, I found it so useful that I simply pasted the full 24-bit colors into a converter at this location in order to get my 565 colors.

http://greekgeeks.net/#maker-tools_convertColor

--bob
